apps/crn-frontend/src/network/teams/ManuscriptToastProvider.tsx

```diff
@@ -15,16 +18,24 @@ export const ManuscriptToastProvider = ({
   children: React.ReactNode;
 }) => {
   const [showSuccessBanner, setShowSuccessBanner] = useState<boolean>(false);
+  const [formType, setFormType] = useState<FormType>('');
+
+  const formTypeMapping = {
+    manuscript: 'Manuscript',
+    'compliance-report': 'Compliance Report',
+  };
 
   return (
-    <ManuscriptToastContext.Provider value={{ setShowSuccessBanner }}>
+    <ManuscriptToastContext.Provider
+      value={{ setShowSuccessBanner, setFormType }}
+    >
       <>
-        {showSuccessBanner && (
+        {showSuccessBanner && !!formType && (
```

Comment:
maybe we could remove this `showSuccessBanner` and check if `formType` is truthy?